src/modules/commander/Arming/HealthFlags/HealthFlags.h

```diff
@@ -43,7 +43,40 @@
 
 #include <px4_platform_common/log.h>
 #include <uORB/topics/vehicle_status.h>
-#include <uORB/topics/subsystem_info.h>
+
+struct subsystem_info_s {
+	// keep in sync with mavlink MAV_SYS_STATUS_SENSOR
+	static constexpr uint64_t SUBSYSTEM_TYPE_GYRO = 1;
+	static constexpr uint64_t SUBSYSTEM_TYPE_ACC = 2;
+	static constexpr uint64_t SUBSYSTEM_TYPE_MAG = 4;
+	static constexpr uint64_t SUBSYSTEM_TYPE_ABSPRESSURE = 8;
+	static constexpr uint64_t SUBSYSTEM_TYPE_DIFFPRESSURE = 16;
```

Comment:
Maybe just a detail but I'd define them like this:
```suggestion
	static constexpr uint64_t SUBSYSTEM_TYPE_GYRO = 1<<0;
	static constexpr uint64_t SUBSYSTEM_TYPE_ACC = 1<<1;
	static constexpr uint64_t SUBSYSTEM_TYPE_MAG = 1<<2;
	static constexpr uint64_t SUBSYSTEM_TYPE_ABSPRESSURE = 1<<3;
	static constexpr uint64_t SUBSYSTEM_TYPE_DIFFPRESSURE = 1<<4;
```
More readable, less error-prone.